packages/gestalt-design-tokens/CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+This file is updated manually
```

Comment:
We should look into updating this file automatically with the PR summary whenever changes are detected within `/packages/gestalt-design-tokens`

Reply:
Yes... I think we should create a tokens package with separate release and create the changelog from there.